ui/tests/e2e/fixtures/environmentsPage.js

```diff
@@ -0,0 +1,221 @@
+import { expect } from '@playwright/test';
+import { ENV } from '../env';
+
+export class EnvironmentPage {
+  /**
+   * @param {import('@playwright/test').Page} page
+   */
+  constructor(page) {
+    this.page = page;
+  }
+  async navigate() {
+    await this.page.goto(`${ENV.MESHERY_SERVER_URL}/management/environments`);
+  }
+  async fillInput(locator, value) {
+    await locator.fill(value);
+  }
+
+  async goToProfiles() {
+    await this.page.goto(`${ENV.MESHERY_SERVER_URL}/management/environments`);
+    await expect(this.page).toHaveURL(`${ENV.MESHERY_SERVER_URL}/management/environments`);
+  }
+
+  async goToConnections() {
+    await this.page.goto(`${ENV.MESHERY_SERVER_URL}/management/connections`);
+    await expect(this.page).toHaveURL(`${ENV.MESHERY_SERVER_URL}/management/connections`);
+  }
+
+  async doesProfileExist(environmentName) {
+    await this.goToProfiles();
+    return this.page
+      .locator(`text=${environmentName}`)
+      .count()
+      .then((count) => count === 1);
+  }
+  async createEnvironmentProfile(environmentName) {
+    if (!(await this.doesProfileExist(environmentName))) {
+      await this.page.getByRole('button', { name: 'Create' }).click();
+      await this.fillInput(this.page.locator('#Environment_name'), `${environmentName}`);
+      await this.fillInput(
+        this.page.locator('#Environment_description'),
+        'Created profile for playwright tests',
+      );
+      await this.page.getByRole('button', { name: 'Save', exact: true }).click();
+      // Check for notification visibility
+      const notification = await this.page.locator(`text=Environment "${environmentName}" created`);
```

Comment:
Here is an example of injecting a test ID on a notification

https://github.com/meshery/meshery/pull/12102/files#diff-fca7e9dcc189b32237c83af054b157e76b2c5d37fc168fef5145ae82441411e3R69

https://github.com/meshery/meshery/pull/12102/files#diff-328c2de500bf9e8980958bf72da5e651b9a226e940906a17cf6f282d58a4c6bbR1229